internal/scraper/scraper.go

```diff
@@ -52,7 +69,7 @@ type Scraper struct {
 	logger        zerolog.Logger
 	check         sm.Check
 	probe         sm.Probe
-	bbeModule     *bbeconfig.Module
+	bbeModule     *ConfigModule
```

Comment:
```suggestion
	module     *ConfigModule
```
? Maybe this is no longer a BBE Module